src/server/services/tips/predicates.test.ts

```diff
@@ -0,0 +1,513 @@
+import MockDate from 'mockdate';
+import { ApiResponse, ApiSuccessResponse } from '../../../universal/helpers';
+import { BRPData, BRPDataFromSource } from '../../../universal/types';
+import BRP from '../../mock-data/json/brp.json';
+import WPI_STADSPAS from '../../mock-data/json/wpi-stadspas.json';
+import WPI_AANVRAGEN from '../../mock-data/json/wpi-aanvragen.json';
+import UITKERINGEN from './mock-data/wpi_aanvragen.mock.json';
+import TONK from './mock-data/tonk.mock.json';
+import TOZO from './mock-data/tozo.mock.json';
+import TOERISTISCHE_VERHUUR from './mock-data/vakantie_verhuur.mock.json';

```

Comment:
Waarom heb je hier opnieuw mock data toegevoegd? / Gebruik je alle data in deze files? Ik vind het in de python tips api super onduidelijk om functionaliteit te testen tegen adhv json files.

Verder, kun je de `-` gebruiken als koppelteken?

Reply:
De data wordt gebruikt om app-states te maken om zo te controleren of predicate functies het juist gedrag vertonen en om in de tips-service test te checken of de juiste tips naar voren komen gegeven een app-state.
Als je suggesties hebt om dit anders te doen, laat maar weten.

Reply:
```
const TONK_CONTENT = WPI_E_AANVRAGEN.content.filter(aanvraag => aanvraag.about === 'TONK');
const TOZO_CONTENT = WPI_E_AANVRAGEN.content.filter(aanvraag => aanvraag.about === 'TOZO');
...etc.
```

De wpi aanvragen zijn ook te importeren. Basically alles uit `../../mock-data/json/*`